invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -89,6 +94,7 @@ const count = query => {
 
 export const document = {
   get: get,
+  del: del,
```

Comment:
`delete`? We had already used it I think in another pr :)

Reply:
Absolutely right!